apps/climatemappedafrica/src/components/HURUmap/IndicatorTitle/index.js

```diff
@@ -61,6 +68,86 @@ function IndicatorTitle({
 }) {
   const classes = useStyles(props);
 
+  const { chartType, geoCode, indicatorId, isCompare } = props;
+
+  const url = new URL(
+    `/embed/${geoCode.toLowerCase()}/${indicatorId}`,
+    site.environmentUrl,
+  ).toString();
+  const className = `wrapper-${geoCode}-${indicatorId}`;
+  const code = `<div>
```

Comment:
Is this code application specific i.e. does it contain anything PesaYetu specific that isn't applicable to say HURUmap?